envm/Dockerfile

```diff
@@ -1,46 +1,325 @@
+      make install ) && \
+    echo "Verifying installation" 1>&2 && \
+      "${R_INSTALL_PATH}/bin/R" --version && \
+    echo "Link installation to /usr/local/bin" 1>&2 && \
+    ln -s "${R_INSTALL_PATH}/bin/R" /usr/local/bin/R && \
+    ln -s "${R_INSTALL_PATH}/bin/Rscript" /usr/local/bin/Rscript && \
+    echo "Capabilities of R are as follows" 1>&2 && \
+      Rscript -e "capabilities()" 1>&2 && \
+    echo "Clean up source" 1>&2 && \
+      rm -rf "R-${R_PATCH_VERSION}" "R-${R_PATCH_VERSION}.tar.gz"
+
+RUN echo "Installing pandoc" 1>&2 && \
+      echo "Got target platform value ${TARGETPLATFORM}" 1>&2 && \
+      if [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
+        pandoc_platform_arg="amd64"; \
+      elif [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+        pandoc_platform_arg="arm64"; \
+      else \
+        echo "Warning target platform $TARGETPLATFORM is not supported" 1>&2 && \
+        echo "Defaulting to linux/amd64" 1>&2 && \
+        pandoc_platform_arg="amd64"; \
+      fi && \
+      DEBIAN_FRONTEND=noninteractive apt-get update -y -qq && \
+      DEBIAN_FRONTEND=noninteractive apt-get install -y -qq \
+        texlive && \
+      wget --no-verbose --output-document "pandoc.deb" \
+      "https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-1-$pandoc_platform_arg.deb" && \
+      DEBIAN_FRONTEND=noninteractive dpkg --install "pandoc.deb" && \
+      rm "pandoc.deb" && \
+    echo "Clean up apt" 1>&2 && \
+      rm -rf "/var/lib/apt/lists"
+
+# Set up conda env file
 COPY ./envm/environment.yaml /tmp/environment.yaml
-RUN conda env create -f /tmp/environment.yaml
 
-# Activate the environment and set path
-ENV PATH  /miniconda/envs/rnasum/bin:$PATH
-ENV CONDA_PREFIX /miniconda/envs/rnasum
+RUN echo "Download and Install conda" 1>&2 && \
+      echo "Got target platform value ${TARGETPLATFORM}" 1>&2 && \
+        if [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
+          mamba_platform_arg="x86_64"; \
+        elif [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+          mamba_platform_arg="aarch64"; \
+        else \
+          echo "Warning target platform $TARGETPLATFORM is not supported" 1>&2 && \
+          echo "Defaulting to linux/amd64" 1>&2 && \
+          mamba_platform_arg="x86_64"; \
+        fi; \
+      wget --no-verbose --output-document mambaforge.sh \
+        "https://github.com/conda-forge/miniforge/releases/download/${MAMBAFORGE_VERSION}/Mambaforge-pypy${PYPY_VERSION}-${MAMBAFORGE_VERSION}-Linux-$mamba_platform_arg.sh" && \
+      bash mambaforge.sh -b -p /miniforge && \
+    echo "Deleting conda install script" 1>&2 && \
+      rm mambaforge.sh && \
+    echo "Update mamba" 1>&2 && \
+      /miniforge/bin/mamba update --quiet --yes \
+        --name base \
+        --channel conda-forge \
+        mamba && \
+    echo "Install environment with mamba" 1>&2 && \
+      /miniforge/bin/mamba env create --quiet \
+        --file /tmp/environment.yaml && \
```

Comment:
Why is mamba being used if we are later using renv to create the environment?

Reply:
There are a couple of dependencies in the conda env that aren't R related which is why this is still there.

```
dependencies:
        - plotly-orca
        - libopenblas==0.3.7
```

One could try install these through other means if we want to remove conda entirely.